src/Indexer.php

```diff
@@ -137,4 +137,9 @@ public function setBulkMaxSize(int $bulkMaxSize): void
             $this->flush();
         }
     }
+
+    public function getBulkMaxSize(): int
+    {
+        return $this->bulkMaxSize;
+    }
```

Comment:
Can you remove this getter? It's not used anymore :+1:

Reply:
done via fef2f6ed8d283f6120c0e5229a7fc057405ff0f9